src/DotLiquid/Util/ObjectExtensionMethods.cs

```diff
@@ -11,6 +11,9 @@ public static bool RespondTo(this object value, string member, bool ensureNoPara
             if (value == null)
                 throw new ArgumentNullException("value");
 
+            if (value is DropBase)

```

Comment:
Late comment inspired by @microalps comment in the open feature request: should test for is IIndexable contract instead, as that's where the ContainsKey method is defined. Same below, as the array accessor is also defined in IIndexable.